.gitattributes

```diff
@@ -1,5 +1,6 @@
 /.github/                       export-ignore
 /.phive/                        export-ignore
+/src/Test/                      export-ignore
```

Comment:
When you work on a package and declare test code in subdirectories of the production code with separate configuration files, adding exclusions for test code in `.gitattributes` is even less work.

There is no need to add configuration files as they are in the excluded directory.